### ISSUE TITLE
Failed nightly build 2024-01-18

### ISSUE THREAD

**Issue by jdangerx** [catalyst-cooperative/pudl#3250]

**Comment by zaneselvans** [comment]
Yes, 95th percentile of gas prices was a little out of bounds, probably because late 2022 had high prices due to Putin.  I've adjusted it upwards slightly and am re-validating locally. Will push a fix as soon as validations pass.
